### ISSUE TITLE
output bbox doesn't map to actual objects

### ISSUE THREAD

**Issue by sujit25** [dongfangduoshou123/YoloV3-TensorRT#21]
I've tried passing in onnx and trt file for yolov3 (416) dimension. Generated bboxes are very small as compared to expected size of bboxes for actual objects in video.

**Comment by dongfangduoshou123** [comment]
The location is right, just the box size is wrong?

**Comment by sujit25** [comment]
Yeah. I will share a snapshot for explaining about issue. I've deserialised existing trt engine file (generated by python scripts given by Nvidia in tensorRT samples). I have debugged existing flow in which trt engine is deserialised and used for Inference. Bounding boxes are present in correct locations but their size is very small. I have found that in code we are already scaling back bboxes. Not sure why they are not mapping to complete bboxes.
